cunumeric/module.py

```diff
@@ -2116,7 +2118,12 @@ def array_split(
         split_pts = list(indices)
         # adding the size of the target dimension.
         # This helps create dummy or last subarray correctly
-        split_pts.append(array.shape[axis])
+        try:
+            split_pts.append(array.shape[axis])
+        except AttributeError:
```

Comment:
actually, last quick question: what exactly can cause an *Attribute* error here? That seems surprising.

Reply:
It seems not be accessible. Remove _attribute_ error here.